kukur/loader.py

```diff
@@ -87,14 +91,14 @@ def open_child(self, name: str):
         If files_as_path is True, return the path of the child.
         """
         if not self.__path.is_dir():
-            raise InvalidDataError(f'"{self.__path}" is not a directory')
```

Comment:
This should also be an `InvalidDataError`: there is a misconfigured source in this case.